content/en/news/kubecon-china-2021.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'KubeSphere Team will join the KueCon China and bring 5 sessions'
+keywords: KubeCon, Meetup, Kubernetes
+description: KubeSphere brings 4 sessions and participte 1 office hour in KubeCon China2021
+createTime: '2021-12-09'
+author: 'Feynman, Lindsay'
+---
+# KubeSphere Team will join the KubeCon China and bring 5 sessions
```

Comment:
Please remove this line: # KubeSphere Team will join the KubeCon China and bring 5 sessions, it is duplicated with the title you defined above.